.travis.yml

```diff
@@ -12,9 +12,9 @@ before_script:
   # - export REDIS_PASS=pass
 
 script:
- - go run examples/00_test/ext/ext.go &
- - go run examples/03_pokemon/ext/ext.go -port 6770 &
- - go test ./fsm ./clf ./bot ./ext -cover -coverprofile=coverage.txt
+  - go run examples/00_test/ext/ext.go &
+  - go run examples/03_pokemon/ext/ext.go -port 6770 &
+  - go test ./... -cover -coverprofile=coverage.txt
```

Comment:
Adding `./...` here ensures we run all the tests. Not sure if you knew about this.